### PR TITLE
Consistent page summary

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Currently, there is an inconsistency in the universe summaries used in the dapp. Some summaries display the title first followed by the logo, while others display the logo before the title. This PR addresses this issue and ensures all summaries have a consistent appearance.
The `UniversePageSummary` component already has the correct logo/title order, so we will reuse it as much as possible to maintain consistency.

Demo: https://qsgjb-riaaa-aaaaa-aaaga-cai.mstr-ingress.devenv.dfinity.network/

# Changes

- Refactoring: The Summary component contained props and extra logic specific to the canisters page. This logic has been extracted and moved directly to the canisters page ([commit](https://github.com/dfinity/nns-dapp/pull/5032/commits/07950611bc66ed7e2255a7e0cfcf414934ed3d86)).
- Component Replacement: The custom title and logo in the Summary component have been replaced with the UniversePageSummary component.
- Component Removal: The SummaryLogo component has been removed as it is no longer in use.
- HTML Compliance: Replaced `<div>` elements with `<span>` elements to ensure the UniversePageSummary component can be rendered inside an `<h1>` tag, complying with HTML standards.

# Tests

- Enhancement: Extended UniversePageSummaryPO with a getLogoAlt function to test the alt attribute of the logo.
- Test Removal: Removed SummaryLogo.spec.ts as the SummaryLogo component has been removed.
- Test Updates: Updated existing tests to ensure coverage is maintained despite the removal of the SummaryLogo tests.

# Todos

- [ ] Add entry to changelog (if necessary).
Not necessary.

# Screenshots

| Before | After |
|--------|--------|
| image | image |
| image | image |
| image | image |
| image | image |
| image | image |

### CODE REVIEW

File: frontend/src/lib/pages/Canisters.svelte [hunk 1]
```diff
@@ -116,11 +122,27 @@
...
+  .summary {
+    display: flex;
+    flex-direction: column;
+    margin: 0 0 var(--padding-3x);
+  }
+
+  .summary-title {
+    display: inline-flex;
+  }
+  .summary-details {
```
Comment (mstrasinskis) [inline frontend/src/lib/pages/Canisters.svelte:140]: Copied/pasted from here https://github.com/dfinity/nns-dapp/pull/5032/files#diff-1e0189f9442a5667996324b8a6a78cb1e4d1a131e4adbe5b7548707e92f65138L55.

File: frontend/src/lib/pages/Canisters.svelte [hunk 2]
```diff
@@ -116,11 +122,27 @@
...
   }
 
+  .summary {
+    display: flex;
+    flex-direction: column;
+    margin: 0 0 var(--padding-3x);
+  }
+
+  .summary-title {
+    display: inline-flex;
```
Comment (mstrasinskis) [inline frontend/src/lib/pages/Canisters.svelte:138]: Moved from https://github.com/dfinity/nns-dapp/pull/5032/files#diff-1e0189f9442a5667996324b8a6a78cb1e4d1a131e4adbe5b7548707e92f65138L43

File: frontend/src/lib/components/universe/UniverseLogo.svelte [hunk 3]
```diff
@@ -12,12 +12,16 @@
...
+  class:horizontalPadding
+  data-tid="project-logo"
+>
   <Logo src={universe.logo} alt={title} {size} {framed} testId="logo" />
-</div>
+</span>
 
 <style lang="scss">
-  div {
+  .container {
```
Comment (dskloetd) [inline frontend/src/lib/components/universe/UniverseLogo.svelte:24]: No change required, but please answer this question.

Is there any functional change in this file?
I also like using a class better than selecting `div` directly, but I'm a bit confused why this is in this PR.

Am also curious why you changed from `div` to `span`.

File: frontend/src/lib/components/universe/UniversePageSummary.svelte [hunk 4]
```diff
@@ -5,10 +5,10 @@
   export let universe: Universe;
 </script>
 
-<div class="summary" data-tid="universe-page-summary-component">
+<span class="summary" data-tid="universe-page-summary-component">
```
Comment (dskloetd) [inline frontend/src/lib/components/universe/UniversePageSummary.svelte:8]: No change required.

Why the change from `div` to `span`?

**Review by dskloetd — APPROVED** [review]
Thanks! Just some questions.

File: frontend/src/lib/components/summary/Summary.svelte
Comment (dskloetd) [inline frontend/src/lib/components/summary/Summary.svelte:1]: Actually, I think this deserved a change log entry, don't you think?

Comment (dskloetd) [reply frontend/src/lib/components/summary/Summary.svelte:1]: I've added one.

File: frontend/src/lib/components/universe/UniversePageSummary.svelte (hunk 4 above)
Comment (mstrasinskis) [reply frontend/src/lib/components/universe/UniversePageSummary.svelte:8]: Because this component is used now inside `<Summary>/H1`, and <div> inside an <h1> tag is not allowed according to HTML standards.

File: frontend/src/lib/components/universe/UniverseLogo.svelte (hunk 3 above)
Comment (mstrasinskis) [reply frontend/src/lib/components/universe/UniverseLogo.svelte:24]: @dskloetd, there is no functional change in this file. The class was added to have cleaner selectors. The tag has been switched from a div to a span to meet HTML standards.
